sunpy/net/dataretriever/sources/noaa.py

```diff
@@ -94,3 +94,52 @@ def _can_handle_query(cls, *query):
             if x.__class__.__name__ == 'Instrument' and x.value == 'noaa-predict':
                 return all(chklist)
         return False
+
+class SRSClient(GenericClient):
+
+    @staticmethod
+    def _get_default_uri():
+        today = datetime.datetime.utcnow()
+        return ['ftp://ftp.swpc.noaa.gov/pub/warehouse/{date:%Y}/SRS/{date:%Y%m%d}SRS.txt'.format(date=today)]
```

Comment:
This URL is only valid for the current year. In previous years all the days are tarballed: ftp://ftp.swpc.noaa.gov/pub/warehouse/2016/